runtime/mswin.vim

```diff
@@ -1,8 +1,8 @@
 " Set options and add mapping such that Vim behaves a lot like MS-Windows
 "
 " Maintainer:	The Vim Project <https://github.com/vim/vim>
-" Last Change:	2023 Aug 10
-" Former Maintainer:	Bram Moolenaar <Bram@vim.org>
+" Last Change:	2024 Jan 3
+" Former Maintainer:	Shixian Li <lsx7@sina.com>
```

Comment:
Why are you changing former maintainer?

Reply:
Would not it be better keeping the "former maintainer" and adding "current maintainer" ?

Reply:
Don't change this line

Reply:
Sorry, I misunderstood the meaning of this sentence.

Reply:
> Don't change this line

@chrisbra 
Excuse me, do you mean don't modify all lines, including dates, or don't modify the author, or instead of deleting but adding a line of author?